lambda/src/index.js

```diff
@@ -151,4 +153,4 @@ function sleep (ms) {
   return new Promise(resolve => setTimeout(resolve, ms))
 }
 
-// exports.handler({}, {})
+exports.handler({}, {})
```

Comment:
Did you mean to uncomment the this line?

Reply:
Nope, didn't mean to do that. Thanks for catching that. Should be commented out again.